orion/pkg/domains/session.go

```diff
@@ -13,12 +13,12 @@ type Session struct {
 	Id        uint
 	CreatedAt time.Time    `db:"created_at"`
 	UpdatedAt time.Time    `db:"updated_at"`
-	DeletedAt sql.NullTime `db:"deleted_at"`
+	DeletedAt sql.NullTime `db:"deleted_at" json:"-"`
 	ClassId   string       `db:"class_id" json:"classId"`
 	StartsAt  time.Time    `db:"starts_at" json:"startsAt"`
 	EndsAt    time.Time    `db:"ends_at" json:"endsAt"`
-	Canceled  bool         `db:"canceled" json:"canceled"`
-	Notes     NullString   `db:"notes" json:"notes"`
```

Comment:
In this case I actually prefer to be explicit here.
I would prefer to know that the Canceled field will turn into `canceled` and `canceled` for both db and json. Guessing what things may turn out to be is kind of a pain in the butt.

Reply:
So which fields do not need a db tag?

Reply:
oh sorry. This is a confusing comment. Ignore what I said. You're right here. "db" tag is not needed because SQL is case-insensitive. So Canceled & canceled are the same thing.